internal/serviceapi/orgserviceaccountapi/resource_test.go

```diff
@@ -103,27 +107,42 @@ func configBasic(orgID, name, description string, roles []string, secretExpiresA
 	rolesHCL := fmt.Sprintf("[%s]", rolesStr)
 	return fmt.Sprintf(`
 		resource "mongodbatlas_org_service_account_api" "test" {
-			org_id                     = %q
-			name                       = %q
-			description                = %q
-			roles                      = %s
-			secret_expires_after_hours = %d
-		}	
+			org_id                     = %[1]q
```

Comment:
can you unskip tests defined for orgserviceaccountsecret? If you search CLOUDP-366217 you will find the 2 skip lines to remove